html/getstarted.html

```diff
@@ -0,0 +1,485 @@
+	<div id="os1" onclick="OSSelect(1)" style="color:white;width:143px;background-color:rgba(208,51,41,0.9)">Linux (Debian)</div>
+	<div id="os2" onclick="OSSelect(2)" style="color:black;width:143px;background-color:rgba(31,54,77,0.3)">Windows</div>
+	<div id="os3" style="color:black;width:100px;background-color:rgba(31,54,77,0.3);text-align:center;padding: 0;margin:0;">
+		<select id="os-select" onclick="OSSelectMore()">
+			<option selected="selected">&raquo;</option>
+		</select>
+	</div>
+
+	<div></div>
+</div>
+<!-- OS-GRID END -->
+
+<!-- MANAGER-GRID START -->
+<div class="grid-manager">
+	<div id="f2"></div>
+
+	<div style="width:200px;color:black">Package</div>
+	<div id="man0" onclick="ManagerSelect(0)" style="color:white;width:107px;background-color:rgba(208,51,41,0.9)"></div>
+	<div id="man1" onclick="ManagerSelect(1)" style="color:black;width:107px;background-color:rgba(31,54,77,0.3)"></div>
+	<div id="man2" onclick="ManagerSelect(2)" style="color:black;width:107px;background-color:rgba(31,51,41,0.3)"></div>
+	<div id="man3" onclick="ManagerSelect(3)" style="color:black;width:107px;background-color:rgba(31,54,77,0.3)"></div>
+	<div id="man4" onclick="ManagerSelect(4)" style="color:black;width:67px;background-color:rgba(31,54,77,0.3);text-align:center;"></div>
+
+	<div></div>
+</div>
+<!-- MANAGER-GRID END -->
+
+<!-- LANGUAGE-GRID START -->
+<div class="grid-language">
+	<div id="f3"></div>
+
+	<div style="width:200px;color:black">Language</div>
+	<div id="lang0" onclick="LanguageSelect(0)" style="color:white;width:125px;background-color:rgba(208,51,41,0.9)">C++</div>
+	<div id="lang1" onclick="LanguageSelect(1)" style="color:black;width:125px;background-color:rgba(31,54,77,0.3)">Python</div>
+	<div id="lang2" onclick="LanguageSelect(2)" style="color:black;width:125px;background-color:rgba(31,54,77,0.3)">Julia</div>
+	<div id="lang3" onclick="LanguageSelect(3)" style="color:black;width:125px;background-color:rgba(31,54,77,0.3)">All</div>
+
+	<div></div>
+</div>
+<!-- LANGUAGE-GRID END -->
+
+<!-- COMMAND-GRID START -->
+<div class="grid-command">
+	<div id="f4"></div>
+
+	<div style="width:200px;color:black">Run this command</div>
+	<div id="cmd-wrapper" style="padding:0;margin:0">
+		<textarea class="editor-output" rows="6" cols="20" spellcheck="false" id="cmd-run"
+			style="background-color:rgba(31,54,77,0.3);color:black">
+		</textarea>
+	</div>
+
+	<div></div>
+</div>
+<!-- COMMAND-GRID END -->
+
+<!-- ASSETS START -->
+<center>
+	<div style="max-width:800px">
+		<h3 id="page-assets" style="text-align:left">Get your assets</h3>
+
+		<p align="justify">Here is a summary of the currently available assets you can use depending on your needs:</p>
+
+		<br>
+		<div class="asset-wrapper">
+			<div>Asset</div>
+			<div>OS</div>
+			<div>Arch</div>
+			<div></div>
+		</div>
+
+		<hr style="height:0.03em;border:none;color:#333;background-color:#333;" />
+
+		<div class="asset-wrapper">
```

Comment:
I see in the CSS that `.asset-wrapper` has `cursor: pointer` but that seems inaccurate since only the last column (of class `.asset-button`) is actually a link.  Do you think we should remove or change that CSS rule?

Reply:
You are right, fixed in https://github.com/mlpack/mlpack.org/pull/26/commits/febbdadc1c87db8c06b290040b7240c9d4394f10.